trustymail/domain.py

```diff
@@ -31,6 +66,9 @@ def __init__(self, domain_name, timeout, smtp_timeout, smtp_localhost, smtp_port
         self.spf = []
         self.dmarc = []
         self.dmarc_policy = None
+        self.dmarc_pct = None
+        self.dmarc_has_aggregate_uri = False
+        self.dmarc_forensic_uri = False
```

Comment:
See comment below on line 343 or `trustymail.py`.  I think this variable goes by a few different names.

Reply:
Another point...it would be useful to actually pull out the aggregate and forensic URIs and store them in `Domain` as well.  I need to do exactly that, so I can add that functionality in a separate pull request if you don't want to.

Reply:
Added :)